packages/icon/src/Icon.spec.tsx

```diff
@@ -56,6 +56,8 @@ describe('packages/Icon/glyphs/', () => {
         });
       }
 
+      // expect statement wrapped in function
+      // eslint-disable-next-line
```

Comment:
Better practice to say which rule you're disabling:
```suggestion
      // eslint-disable-next-line rule-goes-here
```

So you don't disable all eslint rules, just the one.

Docs: https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments